### PR TITLE
[1LP][RFR] Fix test_search_bar (collections)

### PR DESCRIPTION
**Pull request by gshefer**
{{ pytest: cfme/tests/containers/test_search_bar.py -m 'not long_running_env' --use-provider ocp-v1 }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 1215fb7b176ed78a5bfd571534000f046fea5531:

`cfme/tests/containers/test_search_bar.py`:
- :bangbang: Line 33:41: **E241** *multiple spaces after ','*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/containers/test_search_bar.py [hunk 1]
```diff
@@ -18,20 +18,28 @@
...
     This test checks Search bar functionality on every object summary page
     Steps:
         * Goes to <object> page
         * Inserts: Irregular symbol, '*' character, full search string, partial search string
         * Verify proper results
     """
-    for obj in TEST_OBJECTS:
-        view = navigate_to(obj, 'All')
+    for collection in TEST_OBJECTS:
+        view = navigate_to(collection(appliance), 'All')
```
Comment (quarckster) [inline cfme/tests/containers/test_search_bar.py]: **REQUIRED**
```python
COLLECTION_NAMES = [
    'container_replicators';
    'container_projects',
    'container_routes',
    'container_services',
    'container_pods',
    'object_store_containers'
]
...
for collection_name in COLLECTION_NAMES:
    collection = getattr(appliance.collections, collection_name)
    view = navigate_to(collection, 'All')
```

**Comment by quarckster** [comment]
Also, I don't understand why is this test not parametrized.

**Comment by gshefer** [comment]
@quarckster I agree with you about the parametrization, but we are not parametrize it because it's 1 polarion case (all of the objects) and for now we are keeping on unified matching between test items and test cases.

**Comment by quarckster** [comment]
PRT failed.

**Comment by gshefer** [comment]
@quarckster passed

**Pull request merged**